src/validation/error.messages.ts

```diff
@@ -92,10 +92,14 @@ export enum ErrorMessages {
   MAX_CITY_OR_TOWN_LENGTH = "City or town must be 50 characters or less",
   MAX_COUNTY_LENGTH = "County, state, province or region must be 50 characters or less",
   MAX_POSTCODE_LENGTH = "Postcode must be 20 characters or less",
-  MAX_LEGAL_FORM_LENGTH = "Legal form must be 4000 characters or less",
-  MAX_LAW_GOVERNED_LENGTH = "Governing law must be 4000 characters or less",
-  MAX_PUBLIC_REGISTER_NAME_LENGTH = "Name of register must be 4000 characters or less",
-  MAX_PUBLIC_REGISTER_NUMBER_LENGTH = "Registration number must be 32 characters or less",
+  MAX_ENTITY_PUBLIC_REGISTER_NAME_LENGTH = "Name of register must be 4000 characters or less",
```

Comment:
Code change looks fine, but curious as to why these didn't also align to 160 characters, like on the other screens? Might be worth double-checking with analysts that this divergence in implementation is required, unless you've done that already.

Reply:
Thanks Martin, I have already checked with the analyst during the refinement Monday and the the main reason is that they are pointing to different tables. I will send you the document related to it on slack

Reply:
Thanks @mouhajer-ch for checking. As agreed, a bit inconsistent for the user but let's go with this.